### ISSUE TITLE
Adding a new material-like item

### ISSUE THREAD

**Issue by filsdelamour** [filsdelamour/Xenoblade-Chronicles-Wii-Colony-6-Save-Editor#1]
Code from commit c557f13, on 2020/10/01, permits to modify the number of items available.

Adding some new items does not work, even if there is room for them.

For instance, adding two Material items for the `A Gutsy Trader` colony 6 quest (https://xenoblade.fandom.com/wiki/A_Gutsy_Trader), i.e., 'Flier Wing' and 'Flier Straw'. These items are valid, i.e., they appear in the `itemlist.py` list.

When trying to set the required number of such items in order to complete the quest, I get the following message, and no such item is added.

I am surprised that, from the log, item ID is set to None.

I have checked that modifying the number of an already existing item, e.g., 'Small Insect Leg', works fine (cf. log. below).

```bash
bash> python3 XC1SaveEdit.py -s monado02 -c ListItems -f "Flier Wing"
Save time: 2020/10/19 13:1:56
Location: Colony 6
Flier Wing is a valid item name
Collectable: 245
Material: 148
KeyItem: 24
Done
```

```bash
bash> python3 XC1SaveEdit.py -s monado02 -c SetItem -f 'Flier Wing' -n 6
Save time: 2020/10/19 13:1:56
Location: Colony 6
Flier Wing is a valid item name
Collectable: 245
Material: 148
KeyItem: 24
Item Flier Wing with ID None is set to value 6
Done
```

```bash
bash> python3 XC1SaveEdit.py -s monado02 -c ListItems -f "Small Insect Leg"
Save time: 2020/10/19 13:1:56
Location: Colony 6
Small Insect Leg is a valid item name
Collectable: 245
 20  Small Insect Leg
Material: 148
KeyItem: 24
Done
bash> python3 XC1SaveEdit.py -s monado02 -c SetItem -f "Small Insect Leg" -n 21
Save time: 2020/10/19 13:1:56
Location: Colony 6
Small Insect Leg is a valid item name
Collectable: 245
 20  Small Insect Leg
Material: 148
KeyItem: 24
Item Small Insect Leg with ID 2215 is set to value 21
ITEM : CRC FIXED
Done
bash> python3 XC1SaveEdit.py -s monado02 -c ListItems -f "Small Insect Leg"
Save time: 2020/10/19 13:1:56
Location: Colony 6
Small Insect Leg is a valid item name
Collectable: 245
 21  Small Insect Leg
Material: 148
KeyItem: 24
Done
```

**Comment by filsdelamour** [comment]
This behaviour is normal as the sav file, e.g., `monado01` only stores items which are available, i.e., with at least one quantity. Available items slots, over 150 for material, provide a `0` hexadecimal value in order to states that they are free (cf. listItems() or setItem() function, which only show available items when `x != '0'`).
This is also why the listItems() function, without any filter, only displays the list of items with a not null quantity.

In order to add a completely new item, significant code rewrite is necessary.

**Comment by filsdelamour** [comment]
Solved in commit 2395ae0b. Every category can be added.
